frontend/catalyst/compiler.py

```diff
@@ -123,6 +123,13 @@ def run_writing_command(command: List[str], compile_options: Optional[CompileOpt
     ],
 )
 
+MITIGATION_LOWERING_PASS = (
+    "MitigationLoweringPass",
+    [
+        "lower-mitigation",
+    ],
+)
+
 QUANTUM_COMPILATION_PASS = (
     "QuantumCompilationPass",
     [
```

Comment:
Minor: I think this is a good candidate to add to the quantum compilation pipeline.

Reply:
(Not really relevant to your comment but your comment made me think of this): Is mitigation something that is needed to be specified by the user's? Should this be done automatically for all noisy devices? The way it is coded makes me think it is analogous to a template, but maybe it should be more analogous to a compilation pass similar to CSE for example? Maybe with some params. E.g.,

```python
@qml.qnode(..., enable_zne=True, coeff=[1, 2, 3], deg=2)
def foo():
  ...
```

Reply:
Well that's a good question, and mainly one of UI. Romain was also thinking about having it be a global option, but in PennyLane it is a transform you apply to a QNode. So for now we went with the PennyLane way because it was easier (making it global requires a but of additional code), but we could certainly update it in the future.

Reply:
Thank you @dime10  you were faster than me :+1: I will put it in `QUANTUM_COMPILATION_PASS`

Reply:
I like the idea of having it either in the QNode or the qjit as an option in the future :) Lean slightly more to the QNode, just because that feels more like the place to specify quantum-specific compilation options.